packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -154,6 +159,20 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
     }, [appUserRoles]);
 
 
+    /**
```

Comment:
Not a function definition. No need for JSDoc.

```suggestion
    /*
```